### DISCUSSION TITLE
Redis Template을 활용하여 동시성 문제 해결(Redis)

### DISCUSSION THREAD

**Discussion by stopmin** [BDD-CLUB/challenge-every-week-posting#348] [1. 개발 관련 글 모음]
Min의 깜찍한... 아주 깜찍한 간단한.. 영양가 없는 글입니다!
글 쓰는 솜씨가 없구 개인적인 공부용으로 쓴 거라서 재미두 없고 알아보기 힘들지도 모르겟지만 창피하지만 올립니닷 :D

- 레디스
https://velog.io/@stopmin/RedisRemote-Dictionary-Serivce
- 레디스 템플릿을 활용한 락 구현
https://velog.io/@stopmin/%EB%8F%99%EC%8B%9C%EC%84%B1-%EB%AC%B8%EC%A0%9C-RedisTemplate

**Comment by 02ggang9** [comment]
오랜만이네용 재밌게 보겠습니다!
